tests/unit/test_rendering.py

```diff
@@ -113,5 +113,11 @@ def test_form_rendering(self):
         self.assertTrue('Mac OS X 10.13' in rv.data)
         self.assertTrue('http://example.com/' in rv.data)
 
+    def test_wellknown_subpath(self):
+        """Test that the /.wellknown/subpath route gets 404."""
+        rv = self.app.get('/.well-known/{subpath}')
+        self.assertEqual(rv.status_code, 404)
+
```

Comment:
As these tests are about rendering. It would be to check if `subpath` is inside the rendered content.